samples/apps/copilot-chat-app/deploy/README.md

```diff
@@ -13,7 +13,7 @@ This document details how to deploy CopilotChat's required resources to your Azu
 
 # Configure your environment
 Before you get started, make sure you have the following requirements in place:
-- Azure CLI (i.e., az)
+- Azure CLI (i.e., az) (if you already installed Azure CLI, make sure to update your installation)
```

Comment:
nit: mention a recommended version, or just say the latest.

Reply:
fixed it, thx for feedback